gcd/src/test/java/org/spine3/server/storage/datastore/DsCommandStorageShould.java

```diff
@@ -65,13 +60,11 @@ public static void tearDownClass() {
     @Test(expected = NullPointerException.class)
     public void throw_exception_if_try_to_store_null() {
```

Comment:
There isn't, it's an old test from previous datastore storages implementation.
Should I remove it?